seth/client.go

```diff
@@ -483,6 +483,9 @@ func (m *Client) TransferETHFromKey(ctx context.Context, fromKeyNum int, to stri
 
 // WaitMined the same as bind.WaitMined, awaits transaction receipt until timeout
 func (m *Client) WaitMined(ctx context.Context, l zerolog.Logger, b bind.DeployBackend, tx *types.Transaction) (*types.Receipt, error) {
+	l.Info().
```

Comment:
Can it be spammy, we are using it inside our functions, right?

Reply:
yes, but it will run more than once only if gas bumping is enabled.